docs/git/cli/lesson.md

```diff
@@ -0,0 +1,81 @@
+---
+id: lesson
+title: Lesson
+sidebar_label: Lesson
+---
+
+import Feedback from "@theme/Feedback";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Learning Objectives
+
+By the end of this class, you should be able to:
+
+- To _clone_ a repository using Git CLI
+- To _commit_, _fork_ a repository, and _push_ changes using Git CLI
```

Comment:
I think we should have an LO about `git status` too... git-it tells them to use `git status`, but doesn't really explain much about it, but that's probably enough for this class.

Maybe an LO along the lines of:
* See what uncommitted changes you have with `git status`